docs/database/cassandra.md

```diff
@@ -60,15 +60,15 @@ For batch operations, if the individual operations are known to have the same op
 
 **[8]:** If using a port other than the default port for this DBMS and if `server.address` is set.
 
-**[9]:** For batch operations, if the individual operations are known to have the same query text then that query text SHOULD be used, otherwise all of the individual query texts SHOULD be concatenated with separator `; ` or some other database system specific separator if more applicable.
+**[9]:** For sanitization see [Sanitization of `db.query.text`](../../docs/database/database-spans.md#sanitization-of-dbquerytext).
+For batch operations, if the individual operations are known to have the same query text then that query text SHOULD be used, otherwise all of the individual query texts SHOULD be concatenated with separator `; ` or some other database system specific separator if more applicable.
+Even though parameterized query text can potentially have sensitive data, by using a parameterized query the user is giving a strong signal that any sensitive data will be passed as parameter values, and the benefit to observability of capturing the static part of the query text by default outweighs the risk.
 
-**[10]:** SHOULD be collected by default only if there is sanitization that excludes sensitive information.
```

Comment:
This was a note on the `requirement_level`, which got removed here. We have a general note now for the attribute which starts with `For sanitization see Sanitization of db.query.text...`, which starts with the exact same sentence. Having this twice is not needed -> so this is dropped in other md files below as well.

Reply:
Discussed in https://github.com/open-telemetry/semantic-conventions/pull/1100#discussion_r1649060948 - we add it back.